Sources/Utilities/InAppPurchase/InAppPurchaseService.swift

```diff
@@ -117,7 +116,7 @@ extension InAppPurchaseService {
   public func restorePurchases() async -> Result<Void, InAppPurchaseError> {
     do {
       try await AppStore.sync()
-      return .success()
+      return .success(())
```

Comment:
We have a `success()` Result extension that does just this.

Reply:
`PovioKitUtilities` does not depend on `PovioKitCore`. Wanted to make sub-packages have as few dependencies as possible.

Reply:
`Result` extension should be part of the core package 🤔

Reply:
`Result` is part of `Core,` but `InAppPurshaceService` is part of `Utilities`.